go/apps/http_api_nostream/templates/http_api_nostream/edit_help.html

```diff
@@ -58,3 +58,28 @@
        https://go.vumi.org/api/v1/go/http_api_nostream/{{ conversation.key }}/metrics.json \
        -vvv
 </pre></code>
+
+<h1>Retrying failed requests</h1>
+
+<p>When an HTTP request to a <i>Push message URL</i> or <i>Push event URL</i>
```

Comment:
Is it still okay to use `<i>` in HTML, or is it supposed to all be `<em>` (since CSS is supposed to specify italics)?